app/femr/ui/views/history/indexPatient.scala.html

```diff
@@ -97,6 +97,8 @@ <h3 class="pull-left bold margin-top-zero">General Info</h3>
                     <input type="text" class="form-control input-sm pull-left" name="firstName" value="@viewModel.getPatientItem.getFirstName" readonly/>
                     <label class="inputLabel">Last Name:</label>
                     <input type="text" class="form-control input-sm pull-left" name="lastName" value="@viewModel.getPatientItem.getLastName" readonly/>
+                    <label class="inputLabel">Phone Number:</label>
+                    <input type="text" class="form-control input-sm pull-left" name="phoneNumber" value="@viewModel.getPatientItem.getPhoneNumber" readonly/>
```

Comment:
I think this should be `type="tel"`, since then it'll show up as a phone numpad when entering on mobile devices.